src/Api/SecretManagerFeatures/Models/Request/ProjectCreateRequestModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.Entities;
+using Bit.Core.Utilities;
+
+namespace Bit.Api.SecretManagerFeatures.Models.Request
+{
+    public class ProjectCreateRequestModel : IValidatableObject
+    {
+        [Required]
+        public Guid OrganizationId { get; set; }
+
+        [Required]
+        [EncryptedString]
+        public string Name { get; set; }
+
+        public Project ToProject()
+        {
+            return new Project()
+            {
+                OrganizationId = this.OrganizationId,
+                Name = this.Name,
+                DeletedDate = null,
```

Comment:
Do we need to set this here?

Reply:
We don't, good call. Thank you, I removed this line.